### PR TITLE
run flask server and I'm insun's update

### PR DESCRIPTION
**Pull request by leeinsunny**
I add " pip install requests " to README.ME

### CODE REVIEW

File: dt-demo-be/README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ```bash
 (.venv) CapstoneProject % pip install flask
 (.venv) CapstoneProject % pip install flask-cors
```
Comment (doorooful) [inline dt-demo-be/README.md:5]: Can you add pip install pandas as you told so?

**Review by doorooful — CHANGES_REQUESTED** [review]
It'd be better to capture screen shot and share on comments!

File: dt-demo-be/README.md (hunk 1 above)
Comment (leeinsunny) [reply dt-demo-be/README.md:5]: okay~!!

**Comment by leeinsunny** [comment]
Alright~!

**Review by leeinsunny — COMMENTED** [review]
okay!

**Pull request closed without merging**